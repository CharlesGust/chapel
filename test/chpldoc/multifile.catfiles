docs/multifile.txt
docs/second.txt