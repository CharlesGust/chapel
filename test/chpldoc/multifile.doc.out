Module: first
    This module uses another module in a separate file 
   main()
       This function calls the other module's function 

Module: second
   method()
       This method is called by a method in another file 

